py-polars/tests/unit/io/test_spreadsheet.py

```diff
@@ -223,10 +243,26 @@ def test_schema_overrides_11161(excel_file_path: Path) -> None:
         "iter_groups": pl.Float32,
     }
 
+    for workbook_path in (path_xlsx, path_xlsb, path_ods):
+        print(workbook_path)
+        df4 = pl.read_excel(
+            workbook_path,
+            sheet_name="test5",
+            schema_overrides={"dtm": pl.Datetime("ns"), "dt": pl.Date},
+        )
+        assert df4.to_dict(False) == {
+            "dtm": [
+                datetime(1999, 12, 31, 10, 30, 45),
+                datetime(2010, 10, 11, 12, 13, 14),
+            ],
+            "dt": [date(2024, 1, 1), date(2018, 8, 7)],
+            "val": [1.5, -0.5],
+        }
```

Comment:
Can we agree to write test assertions using our `assert_frame_equal` util? At least when working with datetimes/durations with non-standard precision, to make sure dtypes are correct.

Reply:
Done ✌️